.eslint/rules/index.js

```diff
@@ -0,0 +1,19 @@
+/* .eslint/rules/index.js */
+
+const fs = require('fs');
+const path = require('path');
+
+const projectName = 'svg-needs-viewBox';
+
+const ruleFiles = fs.readdirSync('.eslint/rules').filter((file) => file !== 'index.js' && !file.endsWith('test.js'));
+
+const configs = {
+  all: {
+    plugins: [projectName],
+    rules: Object.fromEntries(ruleFiles.map((file) => [`${projectName}/${path.basename(file, '.js')}`, 'error'])),
+  },
+};
+
+const rules = Object.fromEntries(ruleFiles.map((file) => [path.basename(file, '.js'), require('./' + file)]));
```

Comment:
chore: can we add comments above this line and above line 13 that explains what these one-liners are returning?  A link at the top of the file to documentation source would be great too.

Reply:
Just added some comments here! Lemme know what ya think!